spec/models/api/contact_form_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe Api::ContactFormItem, type: :model do
+  describe ".initialize" do
+    it "has corners, departments and grades attributes" do
+      item = described_class.new
+      expect(item).to have_attributes(
+        corners: ContactCorners.to_a,
+        departments: ContactDepartments.to_a,
+        grades: ContactGrades.to_a
```

Comment:
__[RuboCop]__

Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.


<!-- Comment by [SideCI](https://sideci.com) -->